assets/scripts/enable-opt-out.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+(function ($) {
+  // Triggers out-out cookie creation when click on link.
+  $('.core-standards-opt_out-link').on('click', function (e) {
```

Comment:
Minor: The underscore is an alien in the class name; what made you choose it?  `opt_out` should be `opt-out` for consistency.